helm/archive-node/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: archive-node
 description: A Helm chart for Mina Protocol's archive node
 type: application
-version: 0.4.9
+version: 0.4.10
```

Comment:
Hey, can you also update the annotations below in Chart.yaml? The same description used in each CHANGELOG.md works. It's used to associate the version with a changelog in ArtifactHub (e.g. https://artifacthub.io/packages/helm/mina/archive-node?modal=changelog)